app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -47,6 +60,11 @@ export function useCapitalProjectsLayer() {
     autoHighlight: true,
     highlightColor: [129, 230, 217, 218],
     pickable: true,
+    getFilterCategory: (f: Feature<Geometry, CapitalProjectProperties>) => [
+      f.properties.managingAgency,
+      "placeholder",
+    ],
+    filterCategories: filterCategories,
```

Comment:
This can be set to either the acronym list when the query param is `null` (not including any added values like `placeholder`) or an array containing on the selected agency when the query param isn't null. The simpler logic also means we don't need as much logic and intermediate variables in lines 35-49. This is what I set it to:
```
filterCategories:
      managingAgency === null ? fullAgencyAcronymList : [managingAgency],
```